### PR TITLE
using tag for version

### PR DESCRIPTION
**Pull request by chrisdugne**
`Version: v1.2.3.commit`

Screen Shot 2019-05-09 at 5 13 56 PM

### CODE REVIEW

**Comment by chrisdugne** [comment]
[ETQU j'aimerais le numéro de version de l'app et pas le numéro du commit](https://trello.com/c/yAZgXXHu/1275-etqu-jaimerais-le-num%C3%A9ro-de-version-de-lapp-et-pas-le-num%C3%A9ro-du-commit)

**Comment by VincentCATILLON** [comment]
Can you please add a gif ?

**Comment by VincentCATILLON** [comment]
As seen with you, we should concatenate the tag with the commit to be able to compare versions in internal store

File: src/components/version.js [hunk 1]
```diff
@@ -15,7 +15,8 @@ type Props = {|
 const styles = StyleSheet.create({
   version: {
     fontSize: theme.fontSize.extraSmall,
-    textAlign: 'center'
+    textAlign: 'center',
+    paddingBottom: theme.spacing.medium
```
Comment (VincentCATILLON) [inline src/components/version.js]: should be added by home. sorry

**Comment by VincentCATILLON** [comment]
GJ :)

**Pull request merged**